code/game/gamemodes/hivemind/hivemind.dm

```diff
@@ -1,13 +1,15 @@
+GLOBAL_LIST_EMPTY(hivehosts)
+
 /datum/game_mode/hivemind
-	name = "Assimilation"
+	name = "assimilation"
 	config_tag = "hivemind"
 	report_type = "hivemind"
 	antag_flag = ROLE_HIVE
 	false_report_weight = 5
 	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain")
 	restricted_jobs = list("Cyborg","AI")
-	required_players = 24
-	required_enemies = 2
+	required_players = 40
```

Comment:
I'd suggest lowering this to 30, no other modes exceed 30, and there are many players that will immediately latejoin at the start of a round, whereas mode selection only considers readied players.